### DISCUSSION TITLE
关于 ant-design-vue是否支持vue2？

### DISCUSSION THREAD

**Discussion by yujiahan2018** [ant-design/ant-design#39072] [Q&A] [answered]
当vue环境是v2，在使用npm命令 ` npm install ant-design-vue --save` 安装 ant-design时，v1~v3均报下图错误，疑似版本太高，而使用vue3，安装成功。所以想核实一下目前ant-design-vue是否还支持vue2版本
![image](https://user-images.githubusercontent.com/40731243/204435607-18ace27f-ce61-4362-9ffa-73e56ffb3a9f.png)

附环境中vue版本：
![image](https://user-images.githubusercontent.com/40731243/204436134-306d7cae-1c40-408f-ba10-e0f243c8838c.png)

**Comment by zombieJ** [comment, marked as answer]
ant design vue 是由另一个社区维护的哈，到这里反馈哈：https://github.com/vueComponent/ant-design-vue
